### ISSUE TITLE
Implement Placeholders API with relationship and permissions

### ISSUE THREAD

**Issue by niranjan94** [fossasia/open-event-orga-server#3998]
> To be done after #3848 is merged into `nextgen`

This API is for admins to customize all the placeholder images used in the website. 

**Model to use:** `CustomPlaceholder` (Table name: `custom_placeholders`)

**Schema:**
- `id` - autogenerated `integer`
- `name` - `nullable` `string`.
- `original_image_url` - `non-nullable` `string` (Rename existing `url` field)
- `thumbnail_image_url` - `non-nullable` `string` (Read-only. autogenerated like in #3848)
- `large_image_url` - `non-nullable` `string` (Read-only. autogenerated like in #3848)
- `icon_image_url` - `non-nullable` `string` (Read-only. autogenerated like in #3848)
- `copyright` - `nullable` `string`
- `origin` - `nullable` `string`

**Relationships:**
- `one-to-one` with `event-sub-topic` - Optional relationship. May or may not exist. One event-sub-topic cannot have more than one placeholder attached to it. 

**Endpoints & permissions:**
- `GET /v1/placeholders` - Accessible to all. Even without JWT authentication.
- `GET /v1/placeholders/<id>` - Accessible to all. Even without JWT authentication.
- `POST /v1/placeholders` - Accessible only to admin
- `PATCH/DELETE /v1/placeholders/<id>`  - Accessible only to admin
- Relationship endpoint with `event-sub-topic` required too.
